connectOpenAIAPI.php

```diff
@@ -0,0 +1,69 @@
+<?php
+    include("include/init.php");
+    require_once("connectYTAPI.php");
+    include("env_constants.php");
+
+    // call function to retrieve video data from YouTube API
+    $youtubeData = getYoutubeData();
+
+    // array that will hold messages to ask ChatGPT
+    $input = [];
+
+    foreach($youtubeData as $video){
+        $title = $video[0];
```

Comment:
Did we discuss restructuring this `getYoutubeData` so the data is returned like `$video['title']` instead of `$video[0]`? I think that would be easier to work with.